governance/handler.go

```diff
@@ -47,10 +47,10 @@ var (
 
 var GovernanceItems = map[int]check{
 	params.GovernanceMode:          {stringT, checkGovernanceMode, nil},
-	params.GoverningNode:           {addressT, checkAddress, nil},
+	params.GoverningNode:           {addressT, checkAddressOrListOfUniqueAddresses, nil},
```

Comment:
`checkAddressOrListOfUniqueAddresses` should be `checkAddress` , right?

Reply:
Yes, you're right. I restored checkAddress. https://github.com/klaytn/klaytn/pull/1110/commits/7bfb3aedc3fea9ad8c0ff61c74ba98bf338b333f.